packages/labs/ssr-client/package.json

```diff
@@ -33,7 +33,7 @@
     "build:watch": "rollup -c --watch",
     "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
     "build:ts:watch": "tsc --build --watch",
-    "clean": "rm -rf {index,hydrate-support,hydrate}.{js,js.map,d.ts} directives/ development/ test/ *.tsbuildinfo",
+    "clean": "rm -rf {index}.{js,js.map,d.ts} directives/ development/ test/ *.tsbuildinfo",
```

Comment:
Was hydrate there mistakenly before?

Reply:
Yeah, vestigial from when I tried moving them there earlier, missed removing it.